src/pages/console/data/data-model.mdx

```diff
@@ -60,4 +60,20 @@ Applications that use an Amplify GraphQL API have limited feature availability i
 - [Binding UI Components to data](../uibuilder/databinding/), and [generating Collections](../uibuilder/collections/) is not supported
 - [Binding Forms to data, and auto-generated forms](../formbuilder/overview/) are not supported
 
-To get access to these features, turn on DataStore and select a Conflict Resolution strategy.
\ No newline at end of file
+To get access to these features, turn on DataStore and select a Conflict Resolution strategy.
+
+### Unsupported directives and field level parameters
+
+- [Default](/cli/graphql/data-modeling/#assign-default-values-for-fields)
+- [Function](/cli/graphql/custom-business-logic/#lambda-function-resolver)
+- [Http](/cli/graphql/custom-business-logic/#http-resolver)
+- [Predictions](/cli/graphql/connect-to-machine-learning-services/)
+- [PrimaryKey](/cli/graphql/data-modeling/#configure-a-primary-key)
+- [Searchable](/cli/graphql/search-and-result-aggregations/)
+- [MapsTo](/cli/graphql/schema-evolution/)
+- [GroupClaim](cli/graphql/authorization-rules/#user-group-based-data-access)
+- [GroupsField](/cli/graphql/authorization-rules/#field-level-authorization-rules)
```

Comment:
since this isn't a directive, can we call this out as:
```suggestion
- [`@auth` `groupsField` property](/cli/graphql/authorization-rules/#field-level-authorization-rules)
```